### PR TITLE
Amp footer links

### PR DESCRIPTION
**Pull request by NataliaLKB**
Adding the four most clicked links on the AMP pages. Based off of new users. Also corrected a bit of styling.
##### Before:

image
##### After:

image

### CODE REVIEW

File: common/app/views/fragments/footer.scala.html [hunk 1]
```diff
@@ -40,9 +40,34 @@
...
+                        contact us</a></li>
+                    <li class="colophon__item"><a data-link-name="au : footer : vacancies" href="@LinkTo {/info/2014/aug/11/guardian-australia-vacancies}">
+                        vacancies</a></li>
+                }
+                <li class="colophon__item"><a data-link-name="terms" href="@LinkTo {/help/terms-of-service}">
+                    terms &amp; conditions</a></li>
+                @if(currentEdition != Au) {
+                    <li class="colophon__item"><a data-link-name="complaints" href="@LinkTo {/info/complaints-and-corrections}">
+                        complaints &amp; corrections</a></li>
+                }
```
Comment (NataliaLKB) [inline common/app/views/fragments/footer.scala.html]: Aware of the duplication, but thought it was the cleaner way of distinguishing what will display on AMP.

Comment (gtrufitt) [inline common/app/views/fragments/footer.scala.html]: I was adding some links/doing some logic in the footer the other day and was starting to think it definitely re-thinking (maybe more like the main nav) as this is getting kinda soupy! (Not in this PR obviously :D )

Comment (NataliaLKB) [inline common/app/views/fragments/footer.scala.html]: Agreed!

**Comment by NataliaLKB** [comment]
@zeftilldeath take a quick look from a design point of view?

**Comment by zeftilldeath** [comment]
:+1:  Ok great I'll just add the couple of visual tweaks (line height and shifting the chevron by a few pixels) after you have merged.  :+1:

**Comment by johnduffell** [comment]
:+1: looks good! four is my favourite number :grinning:

**Pull request merged**